examples/app.py

```diff
@@ -62,7 +62,7 @@
         cd static
         npm install
         cd ..
-        npm install -g node-sass clean-css requirejs uglify-js
+        npm install -g node-sass@3.8.0 clean-css@3.4.24 requirejs uglify-js
```

Comment:
@lnielsen: Pinnig yields to a successfull build.
`docker-compose up -d` works as well
Unfortunately the loading of datasets fails

> docker-compose run --rm web ./scripts/populate-instance.sh
> Creating database postgresql+psycopg2://invenio3:dbpass123@postgresql:5432/invenio3
> Creating all tables!
>   [####################################]  100%             
> Created all tables!
> User created successfully.
> {'active': True, 'password': '****', 'email': u'info@inveniosoftware.org'}
> 9c410f9c-7d48-406f-a62b-1a4ddc365ef9
> 9c7e11b7-8a9c-4556-810f-31cb043203b5
> 331800e8-89b8-4517-8afe-087c01758763
> 010b5eef-36f4-44ee-97ea-9b2f1e71557d
> b7e60021-853e-4503-b4f7-460f4954573d
> bb439636-64a4-467e-9ebd-c837871286fb
> 67d1ff72-83df-4cb9-8b61-aac17bebe971
> e09c3f4c-cb8a-41ae-86f3-947a9a1f89d6
> cd1ade4e-ef4b-4524-a227-579edd5f6913
> Traceback (most recent call last):
>   File "/home/invenio/.virtualenvs/invenio3/bin/invenio3", line 11, in <module>
>     load_entry_point('invenio3', 'console_scripts', 'invenio3')()
>   File "/home/invenio/.virtualenvs/invenio3/lib/python2.7/site-packages/click/core.py", line 722, in __call__
>     return self.main(*args, **kwargs)
>   File "/home/invenio/.virtualenvs/invenio3/lib/python2.7/site-packages/flask/cli.py", line 374, in main
>     return AppGroup.main(self, *args, **kwargs)
>   File "/home/invenio/.virtualenvs/invenio3/lib/python2.7/site-packages/click/core.py", line 697, in main
>     rv = self.invoke(ctx)
>   File "/home/invenio/.virtualenvs/invenio3/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
>     return _process_result(sub_ctx.command.invoke(sub_ctx))
>   File "/home/invenio/.virtualenvs/invenio3/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
>     return _process_result(sub_ctx.command.invoke(sub_ctx))
>   File "/home/invenio/.virtualenvs/invenio3/lib/python2.7/site-packages/click/core.py", line 895, in invoke
>     return ctx.invoke(self.callback, **ctx.params)
>   File "/home/invenio/.virtualenvs/invenio3/lib/python2.7/site-packages/click/core.py", line 535, in invoke
>     return callback(*args, **kwargs)
>   File "/home/invenio/.virtualenvs/invenio3/lib/python2.7/site-packages/click/decorators.py", line 17, in new_func
>     return f(get_current_context(), *args, **kwargs)
>   File "/home/invenio/.virtualenvs/invenio3/lib/python2.7/site-packages/flask/cli.py", line 251, in decorator
>     return __ctx.invoke(f, *args, **kwargs)
>   File "/home/invenio/.virtualenvs/invenio3/lib/python2.7/site-packages/click/core.py", line 535, in invoke
>     return callback(*args, **kwargs)
>   File "/home/invenio/.virtualenvs/invenio3/lib/python2.7/site-packages/invenio_records/cli.py", line 117, in create
>     click.echo(Record.create(record, id_=id_).id)
>   File "/home/invenio/.virtualenvs/invenio3/lib/python2.7/site-packages/invenio_records/api.py", line 193, in create
>     record.validate(**kwargs)
>   File "/home/invenio/.virtualenvs/invenio3/lib/python2.7/site-packages/invenio_records/api.py", line 140, in validate
>     return _records_state.validate(self, self['$schema'], **kwargs)
>   File "/home/invenio/.virtualenvs/invenio3/lib/python2.7/site-packages/invenio_records/ext.py", line 58, in validate
>     **kwargs
>   File "/home/invenio/.virtualenvs/invenio3/lib/python2.7/site-packages/jsonschema/validators.py", line 541, in validate
>     cls(schema, *args, **kwargs).validate(instance)
>   File "/home/invenio/.virtualenvs/invenio3/lib/python2.7/site-packages/jsonschema/validators.py", line 130, in validate
>     raise error
> jsonschema.exceptions.ValidationError: [u'ALEPH Collaboration'] is not of type u'string'
> 
> Failed validating u'type' in schema[u'allOf'][9][u'properties'][u'added_entry_corporate_name'][u'items'][u'properties'][u'miscellaneous_information']:
>     {u'type': u'string'}
> 
> On instance[u'added_entry_corporate_name'][0][u'miscellaneous_information']:
>     [u'ALEPH Collaboration']
> 

and the webserver gives an internal Server error when trying to search

I do not know whether this is related to the search package problem ?